<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NnrDefinitionDialogBase</class>
 <widget class="QDialog" name="NnrDefinitionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NNR definition</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="3">
    <widget class="QGroupBox" name="groupBoxNNR">
     <property name="toolTip">
      <string>Enable/Disable rule</string>
     </property>
     <property name="title">
      <string>NNR Rule</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2_2_2_2_2_2_2_2_4">
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>Rule name :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEditRuleName">
        <property name="toolTip">
         <string>Rule name...</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Please enter a rule name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2_2_2_2_8">
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>Fail Bin:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="GColorPushButton" name="pushButtonFailBinColor">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Define the Bin color assigned to Good dies NNR outlier</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>314</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel2_2_2_2_2_6">
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>Cluster size:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboBoxNNR_Size">
        <property name="toolTip">
         <string>Die cluster size for NNR  (NRR Candidate Template)</string>
        </property>
        <item>
         <property name="text">
          <string>5 x 5 Dies</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7 x 7 Dies</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9 x 9 Dies</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11 x 11 Dies</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13 x 13 Dies</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15 x 15 Dies</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Whole wafer</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3" colspan="3">
       <widget class="QLabel" name="textLabel2_2_2_2_2_8">
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>Note: NNR Candidate Template size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel2_2_2_2_2_7">
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>Top cluster dies (in %) to use:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxNNR_LA">
        <property name="toolTip">
         <string>Top X% best dies to use in cluster (Location Averaging)</string>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>70.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="3">
       <widget class="QLabel" name="textLabel2_2_2_2_2_9">
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>Note: Location Averaging (L.A.)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel2_2_2_2_2_2_8">
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>NNR threshold:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QComboBox" name="comboBoxNRR_Algorithm">
        <property name="toolTip">
         <string>Select NNR detection threshold algorithm to use</string>
        </property>
        <item>
         <property name="text">
          <string>Mean +/-N*Sigma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Median +/-N*RSigma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IQR: Q1-N*IQR, Q3+N*IQR</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="textLabelNNR_N">
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>N=</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLineEdit" name="lineEditNNR_N">
        <property name="toolTip">
         <string>Custom N factor...</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxFailBin">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>142</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>175</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GColorPushButton</class>
   <extends>QPushButton</extends>
   <header>gcolorpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>doubleSpinBoxNNR_LA</tabstop>
  <tabstop>comboBoxNRR_Algorithm</tabstop>
  <tabstop>lineEditNNR_N</tabstop>
  <tabstop>comboBoxNNR_Size</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
